### PR TITLE
SLE15 add service related rules to HIPAA profile

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add rules for some service checks to SLE15 HIPAA profile

#### Rationale:
Add rules 
- service_cron_enabled
- service_debug-shell_disabled
- service_rexec_disabled
- service_rlogin_disabled
- service_telnet_disabled
- service_xinetd_disabled
- service_zebra_disabled

### CODE REVIEW

File: linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml [hunk 1]
```diff
@@ -14,11 +14,15 @@ rationale: |-
 
 severity: medium
 
+identifiers:
+    cce@sle15: CCE-91437-4
```
Comment (Mab879) [inline linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml:18]:
``` 
cce CCE-91437-4 is included in files: 
 - linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml
 - linux_os/guide/services/cron_and_at/service_crond_enabled/rule.ym
```
Please select a new CCE number for this rule.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Looks good. Just need to change one CCE.

 Thanks!

File: linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml (hunk 1 above)
Comment (teacup-on-rockingchair) [reply linux_os/guide/services/cron_and_at/service_cron_enabled/rule.yml:18]: Thanks @Mab879 should be resolved in 6fb67b1

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
@teacup-on-rockingchair Can you please rebase this pull request on `master`?

**Pull request merged**